nvflare/private/fed/server/training_cmds.py

```diff
@@ -316,17 +319,22 @@ def _process_scope_replies(self, table, conn, replies):
             client_name = engine.get_client_name_from_token(r.client_token)
 
             if r.reply:
-                try:
-                    body = json.loads(r.reply.body)
-                    if isinstance(body, dict):
-                        scope_names = body.get(ScopeInfoKey.SCOPE_NAMES)
-                        default_scope = body.get(ScopeInfoKey.DEFAULT_SCOPE)
-                        self._add_scope_info(table, client_name, scope_names, default_scope)
-                    else:
-                        conn.append_error(f"bad response from client {client_name}: expect dict but got {type(body)}")
-                except BaseException as e:
-                    self.logger.error(f"Bad reply from client: {secure_format_exception(e)}")
-                    conn.append_error(f"bad response from client {client_name}: {secure_format_exception(e)}")
+                if isinstance(r.reply.body, str) and r.reply.body:
+                    self._add_scope_info(table, client_name, r.reply.body, "")
+                else:
+                    try:
+                        body = json.loads(r.reply.body)
```

Comment:
Same situation here as commented above,

Consider the case where r.reply.body = '{"hello": "motor"}'

The old code body going to be a dict {"hello": "motor"}
The new code body will be a string '{"hello": "motor"}'

Reply:
Same as above.